### PR TITLE
UICIRC-593: Add RTL/Jest testing for FinePolicyDetail component in src/settings/FinePolicy

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
Add RTL/Jest testing for FinePolicyDetail component in src/settings/FinePolicy

## Refs
https://issues.folio.org/browse/UICIRC-593

### CODE REVIEW

**Review by Stanislau-Snapok — APPROVED** [review]

File: src/settings/FinePolicy/FinePolicySettings.test.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        data: 'data',
+      };
+      const expectedResult = {
+        data: 'data',
+      };
+
+      expect(normalize(testSchedules)).toEqual(expectedResult);
+    });
+
+    it('should normalize data with passed data', () => {
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/FinePolicy/FinePolicySettings.test.js:133]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,6 +39,7 @@
 * Date format in preview should change depending on localization in `Staff slips`. Refs UICIRC-734.
 * `Preview of patron notice template` title is broken. `Preview of staff slips` title is broken. Refs UICIRC-735.
 * Add RTL/Jest testing for `LostItemFeePolicyDetail` component in `src\settings\LostItemFeePolicy`. Refs UICIRC-621.
+* Add RTL/Jest testing for `FinePolicyDetail` component in `src/settings/FinePolicy`. Refs UICIRC-593.
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply CHANGELOG.md:42]: Fixed

File: src/settings/FinePolicy/FinePolicySettings.test.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  });
+
+  describe('with default props', () => {
+    beforeEach(() => {
+      render(
+        <FinePolicySettings {...testDefaultProps} />
+      );
+    });
+
+    it('should render EntryManager component', () => {
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/FinePolicy/FinePolicySettings.test.js:62]: Fixed

File: src/settings/FinePolicy/FinePolicySettings.test.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+  parseInitialValues,
+} from './FinePolicySettings';
+
+import normalize from './utils/normalize';
+
+const mockDefaultFinePolicyReturnValue = 'testReturnValue';
+
+jest.mock('../wrappers/withPreventDelete', () => jest.fn((component) => component));
+jest.mock('./FinePolicyDetail', () => jest.fn(() => null));
+jest.mock('./FinePolicyForm', () => jest.fn(() => null));
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/FinePolicy/FinePolicySettings.test.js:23]: Fixed

File: src/settings/FinePolicy/FinePolicySettings.test.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+          resources: {
+            finePolicies: {},
+            fixedDueDateSchedules: {},
+          },
+        }
+      ), {});
+    });
+  });
+
+  it('should render EntryManager component', () => {
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/FinePolicy/FinePolicySettings.test.js]: Fixed

**Review by pavelspichonak — APPROVED** [review]

**Pull request merged**